packages/terra-functional-testing/tests/jest/logger/logger.test.js

```diff
@@ -0,0 +1,24 @@
+const Logger = require('../../../lib/logger/logger');
+
+describe('Logger', () => {
+  describe('format', () => {
+    it('should format test with the initialized prefix', async () => {
+      const logger = new Logger({ prefix: 'mock-prefix' });
+
+      expect(logger.format('Example text')).toEqual('[terra-functional-testing:mock-prefix] Example text');
+    });
+  });
+
+  describe('log', () => {
+    it('should log a message to the console', async () => {
+      const logger = new Logger({ prefix: 'mock-prefix' });
+
+      jest.spyOn(console, 'log').mockImplementationOnce(() => { });
+
+      logger.log('Example text');
+
+      expect(console.log).toHaveBeenCalledWith('[terra-functional-testing:mock-prefix] Example text');
```

Comment:
```suggestion
      // eslint-diasable-next-line no-console
      expect(console.log).toHaveBeenCalledWith('[terra-functional-testing:mock-prefix] Example text');
```

Reply:
Added the console log eslint disable:

https://github.com/cerner/terra-toolkit/pull/412/commits/81ec6b8b522dd42261437c12f789f6e373615c9a